xcodeproj/internal/installer.template.sh

```diff
@@ -167,4 +167,9 @@ if [[ -f "$dest/rules_xcodeproj/generated.xcfilelist" ]]; then
   done
 fi
 
+# TODO: Get this approach validated and find the right location to add this, if this approach is approved.
+touch "$dest/rules_xcodeproj/lib"
+DEV_LIB_DIR="$(xcode-select -p)/Toolchains/XcodeDefault.xctoolchain/usr/lib"
+ln -sF "$DEV_LIB_DIR" "$dest/rules_xcodeproj/lib"
```

Comment:
Hey! Yes, after your suggestion of having it in `bazel_build.sh`, I discovered that Xcode expects the symlink to be available before all the build phases are evaluated. So, the options were either to have it in the installer template or in the Scheme's pre-actions.

Exploring the scheme's pre-actions because:

1. We get a lot more control here. We can only apply this only for launchable targets instead of all and also listen to the thread sanitizer env variable.

Please let me know your thoughts on this and we can discuss about how to address this more elegantly.

Reply:
Ahh, I didn't know Xcode checked the symlink before `Bazel Build` ran. Okay, a scheme pre-action works!